Push/PushRegistration.swift

```diff
@@ -30,11 +32,15 @@ public class PushRegistration: NSObject, NSCoding {
     }
 
     @objc public convenience required init?(coder aDecoder: NSCoder) {
-        guard let uaid = aDecoder.decodeObject(forKey: "uaid") as? String,
-            let secret = aDecoder.decodeObject(forKey: "secret") as? String,
-            let subscriptions = aDecoder.decodeObject(forKey: "subscriptions") as? [String: PushSubscription] else {
-                fatalError("Cannot decode registration")
+        guard let uaid = aDecoder.decodeObject(of: NSString.self, forKey: "uaid") as? String,
+            let secret = aDecoder.decodeObject(of: NSString.self, forKey: "secret") as? String,
+            let subscriptions = aDecoder.decodeObject(
+            of: [NSDictionary.self, NSString.self, PushSubscription.self],
+            forKey: "subscriptions"
+            ) as? [String: PushSubscription] else {
+            fatalError("Cannot decode registration")
```

Comment:
nit: could be just git but the indentation seems very off

Reply:
Ran an auto indentation for it, and now looks better!